app/wama.web/Controllers/PatronController.cs

```diff
@@ -23,8 +23,9 @@ public PatronController(IUserAccountService userAccountService, ICheckInService
         }
 
         [HttpGet]
-        public IActionResult Create()
+        public IActionResult Create(string ReceavedId)
```

Comment:
"**ReceavedId**" is supposed to be "**receivedId**".

**Apart** from that, you should rename the variable to "**memberId**", this tells me when reading the code that the parameter is a Member Id.